src/Framing/AmqpValue.cs

```diff
@@ -82,11 +82,14 @@ protected virtual void WriteValue(ByteBuffer buffer, object value)
 
         internal override void EncodeValue(ByteBuffer buffer)
         {
-            var byteBuffer = this.value as ByteBuffer;
-            if (byteBuffer != null)
+            if (this.value is ByteBuffer byteBuffer)
```

Comment:
Some of the projects for the old frameworks require VS 2015 to build. This new syntax is not supported, and we have to keep the old style of casting unfortunately.

Reply:
Got it!